package/sbp_fileio_daemon/sbp_fileio_daemon.mk

```diff
@@ -4,6 +4,8 @@
 #
 ################################################################################
 
+ifeq ($(BR2_PACKAGE_SBP_FILEIO_DAEMON),y)
```

Comment:
That's why this is necessary.  This is to avoid installing the script when the package is disabled.
I copied this pattern from starling_daemon: https://github.com/swift-nav/piksi_buildroot/blob/master/package/starling_daemon/starling_daemon.mk#L7, but that uses `BR2_BUILD_STARLING_DAEMON`.  What is the difference?

Reply:
gotcha, makes sense! I'm not 100% sure what `BUILD` vs `PACKAGE` does but if this avoids the overlay that's what matters